app/controllers/sessions_controller.rb

```diff
@@ -16,6 +14,10 @@ def create
     end
   end
 
+  def logout
+    reset_session
```

Comment:
Is this the best/idiomatic way of doing this?
```For removing objects from the session, you can either assign a single key to nil:

# removes :person from session
session[:person] = nil
or you can remove the entire session with reset_session.
```
Answered my own question! From the Rails Docs